Components/ListSearchResults/Partials/Pagination/_style.scss

```diff
@@ -1,29 +1,38 @@
 .pagination {
-  display: block;
-  margin-top: 40px;
+  margin-top: 20px;
 
-  div {
-    display: block;
-    margin: 10px 2px;
+  // Float clearfix
+  &::after {
+    clear: both;
+    content: '';
+    display: table;
+  }
```

Comment:
I like a shorter fix: Instead of adding an ::after element, just add `overflow: auto;` to the .pagination element.